UnitTests/BraintreePayPalTests/BTPayPalApprovalURLParser_Tests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+@testable import BraintreeCore
+@testable import BraintreePayPal
+
+final class BTPayPalApprovalURLParser_Tests: XCTestCase {
```

Comment:
I think these things are already tested through `BTPayPalClient_Tests`. Will comment on each test below with an example of where they are covered

Reply:
These were merged in from PR #1279 based on this feedback from Tim: https://github.com/braintree/braintree_ios/pull/1274#discussion_r1568922780

Reply:
That's fine by me, removed here: https://github.com/braintree/braintree_ios/pull/1274/commits/fabe9984f68cc600f95d0eee3b46d3a3fbd1df7f